app/scrapers/sql/courses.sql

```diff
@@ -0,0 +1,81 @@
+CREATE TABLE IF NOT EXISTS courses (
+    id SERIAL PRIMARY KEY,
+    course_id text NOT NULL,
+    semester int NOT NULL,
+    name text NOT NULL,
+    dept_id text NOT NULL,
+    department text NOT NULL,
+    credits text NOT NULL,
+    description text,
+    grading_method text[],
+    gen_ed text[],
```

Comment:
I had initially removed CORE, but it is going to be added back, for sake of compatibility